src/main/resources/templates/email/ltft-submitted-tpd/v1.0.0.html

```diff
@@ -31,7 +31,7 @@ <h2>Subject</h2>
     <li>WTE change <th:block th:text="${not #strings.isEmpty(var?.change?.wte)} ? ${var.change.wte} : 'unknown'"></th:block></li>
   </ul>
   <p>If you have any queries regarding this application, <th:block th:with="contact = ${contacts?.get('LTFT_SUPPORT')}">
-      <th:block th:replace="~{fragments/link/v1.0.0 :: link(${contact?.contact}, ${contact?.type}, 'please click on the following link for Local Office Support. ', ${contact?.contact}, '', 'please contact your <strong>Local Office Support team.</strong>', |Ltft Approved - Enquiries - ${subjectRefs}|)}"></th:block>
+      <th:block th:replace="~{fragments/link/v1.0.0 :: link(${contact?.contact}, ${contact?.type}, 'please click on the following link for Local Office Support. ', ${contact?.contact}, '', 'please contact your <strong>Local Office Support team.</strong>', |Ltft Submitted - Enquiries - ${subjectRefs}|)}"></th:block>
```

Comment:
❤️

Reply:
Thank you Reuben.
I am not sure if I get where you mean correctly, but the `ltft-submitted-tpd-full-email-contacts.html ` is also updated, Please advise if somewhere else need updated.

Reply:
No worries, it is just the '`Ltft Approved - Enquiries - ${subjectRefs}`' text. But I'll fix that along with the tabulation.